src/main/java/io/dashboardhub/pipelinedashboard/domain/User.java

```diff
@@ -23,10 +26,10 @@
     private String username;
 
     private String name;
+    private String email;
 
-//    @ManyToMany
-//    @JoinTable(name="users_projects")
-//    private Set<Project> projects;
+    @OneToMany(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
```

Comment:
`EAGER` is probably not right, it was a question I had for you as it would not work if wasn't like this. 

You are right though, may be a **bidirectional** relationship is better, because the user can see their login history and the **admin** can see all login history and for what user